auto_nag/scripts/copy_duplicate_info.py

```diff
@@ -117,6 +117,10 @@ def compare(self, bugs, dups):
 
             dup = dups[dupid]
             bs = utils.get_signatures(info["signature"])
+            if dup["signature"] is None:
+                # some bugs don't have a cf_crash_signature field... it's weird
+                # but it's life...
+                continue
```

Comment:
Should we log a warning?

Reply:
Given that the bug is going to be fixed, let's log a warning so we don't silently ignore the problem if it reappears